app/src/components/main/modal/modalLayout/modalLayout.jsx

```diff
@@ -76,11 +79,14 @@ export class ModalLayout extends Component {
     this.setState({ shown: true });
   }
   onKeydown = (e) => {
+    const { okButton, customButton } = this.props;
     if (e.keyCode === 27) {
       this.closeModal();
     }
     if ((e.ctrlKey && e.keyCode === 13) || (e.metaKey && e.keyCode === 13)) {
-      this.onClickOk();
+      customButton && customButton.onClick && customButton.onClick(this.closeModalWithOk);
```

Comment:
Please don't trigger ALL handlers in this case. Let OK button be default handler for the 'enter' key.

Reply:
Fixed